src/gui/transferlistfilterswidget.cpp

```diff
@@ -287,10 +287,10 @@ TrackerFiltersList::TrackerFiltersList(QWidget *parent, TransferListWidget *tran
     noTracker->setData(Qt::DecorationRole, UIThemeManager::instance()->getIcon("network-server"));
     auto *errorTracker = new QListWidgetItem(this);
     errorTracker->setData(Qt::DisplayRole, tr("Error (0)"));
-    errorTracker->setData(Qt::DecorationRole, style()->standardIcon(QStyle::SP_MessageBoxCritical));
+    errorTracker->setData(Qt::DecorationRole, UIThemeManager::instance()->getIcon("error"));
```

Comment:
error.svg should be in `qBt-theme` folder, not `skin` otherwise this won't work.

Reply:
Won't this change cause problem for other icons?

Reply:
Done.

Reply:
You are right... you could undo what i suggested and change this line to
```suggestion
    errorTracker->setData(Qt::DecorationRole, QIcon(QLatin1String(":/icons/skin/ratio.svg")));
```
But i don't know if this will be acceptable. Better leave this comment unresolved until someone else confirms if this is the proper way.

Personally i don't understand why there are 2 folders. qBt-theme and skin. The icons contained in each seem randomly divided to me.

Reply:
Exactly my thought. Unifying them will actually be a really good solution. As I am not a programmer, I thought It may serve some special purposes.
@FranciscoPombal @Chocobo1 What do you guys think?

Reply:
Maybe some historical reason? No idea if it is important or not. Differing to @Chocobo1

Reply:
I think the qbt-theme icons can be replaced with system icons on unix systems that's why there are rwo folders

Reply:
@jagannatharjun I don't think so. It is the icon name that matter not the folder they are in.

Reply:
https://github.com/qbittorrent/qBittorrent/blob/4b409700b0809f82b8aa853b5b936623100bc22a/src/base/iconprovider.cpp#L61

haven't you noticed almost all system  (or qbt-theme folders') icons are wrapped in `getIcon` call

Reply:
@jagannatharjun You misunderstood me. I said in order to use icon-theme used by the Linux Operating System, I don't think the folder name doesn't matter. It is the real icon name like `error.svg` plays the most part.

Reply:
@thalieht I will make the changes as soon as I get an instruction from @Chocobo1. Until then I think changing the folder is a better solution than changing the code.

Reply:
>Maybe some historical reason? 

Definitely some history in there, but I don't recall the details.

I'm not really sure what you guys are asking... about the suggested changes [here](https://github.com/qbittorrent/qBittorrent/pull/12169#discussion_r402402428), I don't think it is future-proof. There is some theming work going on and that change won't fit well, it would be better to load the icons via `UIThemeManager`.

I would also suggest less changes to the .cpp code in this PR as it gets complicated and it is easy to overlook something. If changes to .cpp are required it would be best to do it in a follow-up PR.

Reply:
@Chocobo1 Good point. I recommend unifying these icon in a single folder in future.
@thalieht I have found where `error.svg` is being used, by running following commands. How can we solve this now, following Mike's suggestion? 

```
grep -rnw -e 'error.svg'
transferlistfilterswidget.cpp:200:    errored->setData(Qt::DecorationRole, QIcon(":/icons/skin/error.svg"));
transferlistmodel.cpp:754:    static QIcon cached = QIcon(":/icons/skin/error.svg");

```

```
grep -rnw -e 'getIcon("error")'
transferlistfilterswidget.cpp:290:    errorTracker->setData(Qt::DecorationRole, UIThemeManager::instance()->getIcon("error"));
```

Reply:
@now-im

I would now recommend just focusing on the icons themselves for this PR. That's done and can be delivered.

If some icons keep strange names like "document-edit-verify" or if they are split across folders seemingly arbitrarily, that's something for another issue report/PR in the future, separate from this one.

Reply:
@FranciscoPombal I just want to fix just `error.svg` icon issue. With that this PR will be complete.

Reply:
I have not been following the theming PRs so i don't know how they will play out. I think @Chocobo1 is right. Better keep changes in .cpp to a minimum in this PR (any renamed icons) and if i'm not mistaken this is the only thing remaining. Keep the new icons in the PR but undo any changes in this line, this has been a problem for a long time so i suppose it doesn't matter if it stays a little more.

Such GUI changes have to go through @sledgehammer999 but he is busy most of the time so i hope you won't be discouraged if it takes a while. 
Thanks for all this work.

Reply:
@thalieht I haven't changed anything in .cpp file regarding `error.svg`. I just shifted it from `skin` folder to `qbt-theme`. What should I do now?

Reply:
Change this line back and put it back in `skin` because as you said, it breaks the icon in other places.

Reply:
You should probably also revert the change a few lines below for the warning icon.

Reply:
@thalieht Reverted changes.